examples/experimental/dagster-airlift/dagster_airlift_tests/unit_tests/core_tests/test_load_defs.py

```diff
@@ -425,3 +426,25 @@ def an_asset() -> None: ...
         {"dag_id": "dag1", "task_id": "task1"},
         {"dag_id": "dag2", "task_id": "task2"},
     ]
+
+
+def test_automapped_build() -> None:
+    airflow_instance = make_instance(
+        dag_and_task_structure={"dag1": ["task1", "task2", "standalone"]},
+        task_deps={"task1": ["task2"]},
+    )
+    defs = build_full_automapped_dags_from_airflow_instance(
+        airflow_instance=airflow_instance,
+    )
+
+    dag1_task1 = key_for_task_asset("dag1", "task1")
+    dag1_task2 = key_for_task_asset("dag1", "task2")
+    dag1_standalone = key_for_task_asset("dag1", "standalone")
+
+    specs = {spec.key: spec for spec in defs.get_all_asset_specs()}
+
+    assert specs[dag1_task1].deps == []
+    assert specs[dag1_task2].deps == [AssetDep(dag1_task1)]
+    assert specs[dag1_standalone].deps == []
```

Comment:
[1] I'd expect some assertions around the full dag asset as well, namely that it has a dep on all of these